silx/gui/plot/tools/profile/rois.py

```diff
@@ -232,6 +233,7 @@ def createProfile2(currentData):
                 rgba = rgba.astype(numpy.float)
             currentData = 0.21 * rgba[..., 0] + 0.72 * rgba[..., 1] + 0.07 * rgba[..., 2]
 
+        yLabel = "%s, width = %s" % (str(method).capitalize(), lineWidth)
```

Comment:
I don't think is needed to have the width in the label. For example, in the case of the "free" line, it already appears in the title.

Reply:
But that's not the case on the image i provided below.